linter.py

```diff
@@ -1,41 +1,17 @@
-#
-# linter.py
-# Linter for SublimeLinter3, a code checking framework for Sublime Text 3
-#
-# Written by Adam Altmejd
-# Copyright (c) 2013 Adam Altmejd
-# License: MIT
-#
-# - Todo: make chktex ignore R code blocks in Knitr/Sweave environments instead
-#   of linting scopes separately.
-
-"""Uses chktex to lint LaTeX files."""
-
 from SublimeLinter.lint import PythonLinter, util
 
 
 class Chktex(PythonLinter):
-    """Provides an interface to use chktex in SublimeText with SublimeLinter3."""
-
-    syntax = ('latex', 'latexing', 'latex (knitr)', 'knitr-rnw', 'latex beamer', 'latexing beamer')
-    selectors = {
-        'latex (knitr)': 'text.tex.latex.knitr.ing - meta.block.parameters.knitr - source.r.embedded.knitr',
-        'knitr-rnw': 'text.tex.latex.knitr - meta.block.parameters.knitr - source.r.embedded.knitr'
-    }
-
     cmd = 'chktex -wall "-f%l:%c %k %k %n: %m\n"'
     error_stream = util.STREAM_STDOUT
     regex = (
         r'^(?P<line>\d+):(?P<col>\d+) '
         r'(?:(?P<error>Error)|(?P<warning>Warning)) '
         r'(?P<message>.+)'
     )
-
-    config_file = ('--localrc', 'chktexrc')
     defaults = {
+        'selector': 'text.tex.latex',
```

Comment:
Isn't this just `text.tex.latex  - meta.block.parameters.knitr - source.r.embedded.knitr`?

Otherwise LGTM

Reply:
Hm, good point. Could it be...

```
text.tex.latex, text.tex.latex.knitr - meta.block.parameters.knitr - source.r.embedded.knitr
```

❓ 

Damn, now I need to go and find out how this chktex thing works. Or guess and see if it breaks for someone.

Reply:
No, then it will match the 'wider'(?) region. E.g. if you select `source, source.python - comment` it will select the whole file.

Reply:
yeah, makes sense. Let's just go with your selector.